pkgs/development/python-modules/pxml/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, pythonAtLeast
+, buildPythonPackage
+, fetchPypi
+, blessings
+, six
+, nose
+, coverage
+}:
+ buildPythonPackage rec {
+  pname = "pxml";
+  version = "0.2.13";
+  disabled = pythonAtLeast "3.7";
```

Comment:
I think pythonAtLeast is inclusive, so you need `pythonAtLeast "3.8"` to enable the package on 3.7 but not 3.8

Reply:
oops, yes